services/skus/datastore.go

```diff
@@ -950,24 +950,40 @@ type TimeAwareSubIssuedCreds struct {
 	RequestID    string                    `json:"-" db:"request_id"`
 }
 
-func (pg *Postgres) AreTimeLimitedV2CredsSubmitted(ctx context.Context, blindedCreds ...string) (bool, error) {
+type AreTimeLimitedV2CredsSubmittedResult struct {
+	AlreadySubmitted bool `db:"already_submitted"`
+	Mismatch         bool `db:"mismatch"`
+}
+
+func (pg *Postgres) AreTimeLimitedV2CredsSubmitted(ctx context.Context, requestID uuid.UUID, blindedCreds ...string) (*AreTimeLimitedV2CredsSubmittedResult, error) {
+	return areTimeLimitedV2CredsSubmitted(ctx, pg.RawDB(), requestID, blindedCreds...)
+}
+
+type getContext interface {
+	GetContext(ctx context.Context, dest interface{}, query string, args ...interface{}) error
+}
+
+func areTimeLimitedV2CredsSubmitted(ctx context.Context, dbi getContext, requestID uuid.UUID, blindedCreds ...string) (*AreTimeLimitedV2CredsSubmittedResult, error) {
 	if len(blindedCreds) < 1 {
-		return false, errors.New("invalid parameter to tlv2 creds signed")
+		return nil, errors.New("invalid parameter to tlv2 creds signed")
 	}
 
+	var result = AreTimeLimitedV2CredsSubmittedResult{}
```

Comment:
suggestion(non-blocking): can we declare this closer to where it is [first used](https://github.com/brave-intl/bat-go/pull/2331/files#diff-86884800d41e3e1ec4019ea4d49e370d00ecc0b8235be9ccc8762336847e0afcR981). 

```
var result AreTimeLimitedV2CredsSubmittedResult
err := dbi.GetContext(ctx, &result, query, blindedCreds[0], requestID)
```

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/2331/commits/5a94f9a2ca0398e6432daf4a719d779446ba50dc